registration-processor/core-processor/registration-processor-abis-middleware-stage/src/main/java/io/mosip/registartion/processor/abis/middleware/stage/AbisMiddleWareStage.java

```diff
@@ -607,18 +606,18 @@ private AbisResponseEntity convertAbisResponseDtoToAbisResponseEntity(AbisRespon
 	 * @param key
 	 * @return
 	 */
-	private static String getFaliureReason(Integer key) {
+	private static String getFaliureReason(String key) {
 		if (key == null)
 			return null;
 
-		Map<Integer, String> failureReason = new HashMap<>();
-		failureReason.put(1, "Internal error - Unknown");
-		failureReason.put(2, "Aborted");
-		failureReason.put(3, "Unexpected error - Unable to access biometric data");
-		failureReason.put(4, "Unable to serve the request");
-		failureReason.put(5, "Invalid request / Missing mandatory fields");
-		failureReason.put(6, "Unauthorized Access");
-		failureReason.put(7, "Unable to fetch biometric details");
+		Map<String, String> failureReason = new HashMap<>();

```

Comment:
can you move this to an enum, It should be constant like -
public enum FailureReason {

	INTERNAL_ERROR("1", "Internal error - Unknown");

	private String code;
	private String text;

	private FailureReason(String code, String text) {
		this.code = code;
		this.text = text;
	}

	public String getCode() {
		return code;
	}

	public String getText() {
		return text;
	}
}